server/views/pages/pagesMacros.njk

```diff
@@ -16,9 +14,9 @@
       <div class="govuk-summary-list__row">
             <dt class="govuk-summary-list__key"> {{key}} </dt>
             <dd class="govuk-summary-list__value"> 
-              {% if not value or regExp.test(value) %} 
+              {% if not value or value.length  == 0 %} 
                 &ndash;  
-              {% elif key === "Evidence bagged and tagged" %} 
+              {% elif key === "Evidence bagged and tagged" and value !== "No" %}
```

Comment:
guess - this won't be necessary once we fix the config. We should make a note to remove this then...